### DISCUSSION TITLE
[Bug Report] ERR_CONNECTION_REFUSED on develop

### DISCUSSION THREAD

**Discussion by Emilaia** [stashapp/stash#4552] [Q&A]
**Describe the bug**
I've been using docker for stash for a while now, and love it so far, but for the last couple days I've been unable to enter stash.
My initial suspicion has to do with the fact I auto update the image while on the develop container, since that's basically asking for trouble at times, but I've yet to see a report on it so was wondering if I was the only one or not.
Was there a relevant change since the last update that could've led to something I need to exclude on firewall or something?

**To Reproduce**
1. Have the latest develop branch build on docker on a separate device (let's say it's 192.168.1.100).
2. Try to enter via 192.168.1.100:9999
3. You get an ERR_CONNECTION_REFUSED 

**Additional context**
Same issue regardless of if I reset the device, ensure am using the same network, or different devices.
Again, if it's just a me issue, all the better, but just wanted to make sure first.

**Comment by MrX292** [comment]
there was an update that Reverts #3069 not sure if that was your problem
